outwatch/src/main/scala/outwatch/dom/DomTypes.scala

```diff
@@ -103,9 +103,11 @@ trait HtmlAttrs
     new PropBuilder(key, codec.encode)
 
   // super.className.accum(" ") would have been nicer, but we can't do super.className on a lazy val
-  override lazy val className = new AccumAttrBuilder[String]("class",
-    reflectedAttr(attrKey = "class", propKey = "className", attrCodec = codecs.StringAsIsCodec, propCodec = codecs.StringAsIsCodec),
-    _ + " " + _
+  override lazy val className = new AccumAttrBuilder[String](
+    "class",
+    reflectedAttr(attrKey = "class", propKey = "className", attrCodec = codecs.StringAsIsCodec, propCodec = codecs.StringAsIsCodec), {
+      case (v1, v2) => s"$v1 $v2"
```

Comment:
This might be controversial and normally I like the `PartialFunction` for Tuple better, but here I would prefer to keep the old version. To me, it is more readable.